EasyVCR/Censors.cs

```diff
@@ -204,7 +225,57 @@ internal string CensorBodyParameters(string body, ContentType? contentType)
             return $"{uri.GetLeftPart(UriPartial.Path)}?{ToQueryString(censoredQueryParameters)}";
         }
 
-        private List<object> ApplyBodyCensors(List<object> list)
+        public static string CensorJsonData(string body, string censorText, IReadOnlyCollection<CensorElement> elementsToCensors)
```

Comment:
This function needs a docstring.

Please add docstrings to below functions as well